app/views/components/_autocomplete.html.erb

```diff
@@ -18,10 +18,6 @@
       <span class="govuk-hint app-c-autocomplete__multiselect-instructions">To select multiple items in a list, hold down Ctrl (PC) or Command (Mac) key.</span>
     <% end %>
 
-    <% if hint %>
```

Comment:
I think we must keep the hint before the select element, as we do with the other form components and it seems like we're using it like this in the tags page.